bcipy/helpers/stimuli.py

```diff
@@ -314,24 +323,24 @@ def __call__(self,
             # Note that this is a workaround to the issue of having different number of targetness in
             # each symbol. If a target symbol is prompted more than once, the data is appended to the dict as a list.
             # Which is why we need to convert it to a (np.ndarray) and flatten the dimensions.
-            # This is not ideal, but it works for now.
 
-        # return np.stack(reshaped_data, 0), labels
         return data_by_targets
 
     @staticmethod
-    def centralize_all_data(data, symbol_pos):
+    def centralize_all_data(data, symbol_pos) -> np.ndarray:
```

Comment:
Looks like we need types for the parameters too.

Reply:
Added types for the input parameters.